sdk/core/azure-core/src/main/java/com/azure/core/implementation/JsonPatchOperation.java

```diff
@@ -104,31 +106,11 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        return toJson(new StringBuilder()).toString();
-    }
-
-    @Override
-    public StringBuilder toJson(StringBuilder stringBuilder) {
-        stringBuilder.append("{\"op\":\"")
-            .append(op.toString())
-            .append("\"");
-
-        if (from != null) {
-            stringBuilder.append(",\"from\":\"")
-                .append(from)
-                .append("\"");
-        }
-
-        stringBuilder.append(",\"path\":\"")
-            .append(path)
-            .append("\"");
-
-        if (value.isInitialized()) {
-            stringBuilder.append(",\"value\":")
-                .append(value.getValue());
-        }
+        AccessibleByteArrayOutputStream outputStream = new AccessibleByteArrayOutputStream();
+        JsonWriter writer = DefaultJsonWriter.toStream(outputStream);
+        toJson(writer);
```

Comment:
for the future: would it make sense to add convenience on DefaultJsonWriter so it can hide stream manipulations?

```java
DefaultJsonWriter writer = new DefaultJsonWriter(); 
toJson(writer);
return writer.toString(); // can throw on the wrong state
```